docs/Getting-Started-Training.md

```diff
@@ -176,3 +176,45 @@ start training from scratch.
 2. Resume from a WandB run: if you set `--trainer.wandb.resume`, it will resume the corresponding WandB run with the ID `asdf1234`. You can
 find the WandB run ID in the URL of your WandB run page. For more information, please refer to the
 [WandB documentation](https://docs.wandb.ai/guides/runs/resuming).
+
+## Custom Evaluation Plugins
+
+Levanter supports custom evaluation plugins that run during training to track specialized metrics. Here is an example of a simple perplexity plugin:
+
+```python
+# my_eval_plugin.py
+import jax.numpy as jnp
+import levanter
+from levanter.callbacks import StepInfo
+from levanter.utils.hf_utils import HfTokenizer
+from levanter.eval import EvalPlugin
+
+class PerplexityPlugin(EvalPlugin):
```

Comment:
really we should just have full callback plugins but this is a good starting point. evals are a bit special